src/Npgsql/NpgsqlConnection.cs

```diff
@@ -1509,19 +1509,19 @@ public static void UnmapEnumGlobally<TEnum>(string? pgName = null, INpgsqlNameTr
     /// Maps a CLR type to a PostgreSQL composite type for use with this connection.
     /// </summary>
     /// <remarks>
-    /// CLR fields and properties by string to PostgreSQL enum labels.
+    /// CLR fields and properties are mapped by name to PostgreSQL composite type attributes.
```

Comment:
I don't think PG generally uses "attribute" to refer to the fields of composite types ([docs](https://www.postgresql.org/docs/current/rowtypes.html)) - changing to "field" to be consistent.

```suggestion
    /// CLR fields and properties are mapped by name to PostgreSQL composite type fields.
```